extensions/amp-carousel/0.1/carousel.js

```diff
@@ -79,14 +100,15 @@ export class AmpCarousel extends BaseCarousel {
   /** @override */
   goCallback(dir, animate) {
     const newPos = this.nextPos_(this.pos_, dir);
+    const container = dev().assertElement(this.container_);
```

Comment:
this.container_ is nullable since its null in the constructor and only available starting in buildCallback. to comply with the type signature for animate which doesn't allow for null we call `assertElement` which makes the type here be not nullable and since we use `dev().assert` it will be eliminated in the prod binary.